gnews/gnews.py

```diff
@@ -259,8 +259,11 @@ def get_news_by_topic(self, topic: str):
         if topic in TOPICS:
             query = '/headlines/section/topic/' + topic + '?'
             return self._get_news(query)
+        elif topic in SECTIONS.keys():
```

Comment:
Hey @v-cardona, Thanks for the PR, great work here. can you please write test cases for this, also possible to add example code for this ?